### ISSUE TITLE
Feature request: Add support for basic auth in nats/nats helm chart

### ISSUE THREAD

**Issue by richerlariviere** [nats-io/k8s#97]
I would like to be able to use basic authentication with nats/nats helm chart in a way like nats-operator helm chart. We have an application using NATS but only with user/password authentication.

**Comment by matthiashanel** [comment]
@richerlariviere So what you are asking for is the ability to provide the helm chart with all desired user/passwords?
And have the ability to do that for different accounts? 
And provide permissions?

**Comment by richerlariviere** [comment]
Hi @matthiashanel,

If we take [nats-operator helm chart](https://github.com/nats-io/nats-operator/blob/master/helm/nats-operator/templates/secret.yaml) as an example, we can see that it's possible to feed a secret to Nats-operator containing a client-auth.json ([template](https://github.com/nats-io/nats-operator/blob/master/helm/nats-operator/config/client-auth.json))

I would like to add a config like that:
```
{
  "users": [
    {
      "username": "",
      "password": "",
      "permissions": {}
    }
}

```

**Comment by wallyqs** [comment]
Fixed via #134
